src/main/resources/static/views/interest/Interests.html

```diff
@@ -1,196 +1,24 @@
-                        </br>
-                        <p class="media-comment">
-                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nisi. Nulla quis sem at nibh
-                            elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper
-                            porta.
-                        </p>
-                        <a class="btn btn-info btn-circle text-uppercase" href="#" id="Contact"><span
-                                class="glyphicon glyphicon-share-alt"></span> Contact</a>
-                    </div>
-                </div>
-            </li>
-            <li class="media">
-                <a class="pull-left" href="#">
-                    <img class="media-object img-circle"
-                         src="https://s3.amazonaws.com/uifaces/faces/twitter/lady_katherine/128.jpg" alt="profile">
-                </a>
-                <div class="media-body">
-                    <div class="well well-lg">
-                        <h4 class="media-heading text-uppercase reviews">Marco </h4>
-                        <ul class="media-date reviews list-inline">
-                            <li class="aaaa"><em>General Manager at Microsoft</em></li>
-                        </ul>
-                        </br>
-                        <p class="media-comment">
-                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed nisi. Nulla quis sem at nibh
-                            elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper
-                            porta.
-                        </p>
-                        <a class="btn btn-info btn-circle text-uppercase" href="#" id="Contact"><span
-                                class="glyphicon glyphicon-share-alt"></span> Contact</a>
-                    </div>
-                </div>
-            </li>
-        </ul>
-    </div>
-</div>
-<div class="container">
-    <div class="row centered">
-        <ul class="pagination">
-            <li class="disabled"><a href="#">«</a></li>
-            <li class="active"><a href="#">1 <span class="sr-only">(current)</span></a></li>
-            <li><a href="#">2</a></li>
-            <li><a href="#">3</a></li>
-            <li><a href="#">4</a></li>
-            <li><a href="#">5</a></li>
-            <li><a href="#">»</a></li>
-        </ul>
-    </div>
-</div>
-<div ng-include="'views/main/Footer.html'"></div>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<div class="container" ng-init="getAllInvInterest()">
+    <div class="row">
+        <div ng-repeat="interest in interests">
+            <div class="container col-md-4 col-xs-12">
+                <div class="serviceBox">
+                    <div class="service-icon">
+                        <span><i class="fa fa-user-o" aria-hidden="true"></i></span>

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)